actions/setup-wp-env/action.yml

```
@@ -25,9 +25,13 @@ inputs:
         required: false
         default: ''
     config:
-        description: A list of PHP constants that'll be passed to the WordPress container, separated by commas or new lines (e.g. `WP_DEBUG:1, SCRIPT_DEBUG:1`)
+        description: A list of PHP constants that'll be passed to the WordPress container, separated by commas or new lines (e.g. `ELEMENTOR_SHOW_HIDDEN_EXPERIMENTS:1, SOME_CONST:some string`)
```

```suggestion
        description: A list of PHP constants that'll be passed to the WordPress container, separated by commas or new lines (e.g. `WP_MEMORY_LIMIT: 64M, WPLANG: en_US`)
```

maybe?
just to have real-world scenarios, and not to mention elementor